lib/modules/modules.go

```diff
@@ -293,6 +293,10 @@ type Modules interface {
 	SetFeatures(Features)
 	// BuildType returns build type (OSS, Community or Enterprise)
 	BuildType() string
+	// IsEnterpriseBuild returns false for [defaultModules].
+	IsEnterpriseBuild() bool
+	// IsOSSBuild returns true for [defaultModules].
```

Comment:
I think you changed a little too much here, these were fine.

Reply:
Oh, you're right. this is the interface. will revert, thanks